### DISCUSSION TITLE
Bulk edit database fields

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#12325] [Feature Requests]
Is there a way to bulk edit database only fields when applying directus over an existing DB ? 

While I have not tested it yet, one possible way to get this done might be, 
1. Export database schema
2. Edit the fields manually such as interfaces
3. Apply the updated schema

Or is there a nocode way to do the same ?

**Comment by rijkvanzanten** [comment]
I'm not exactly sure what you mean by _bulk edit database only fields_. Do you mean you'd like to set multiple fields to the same interface/options at the same time or?

**Reply by VaZark** [reply]
Yup. Something along those lines.

I'm dealing with tables with columns with an avg of 30-40 fields. So in order to onboard the fields of a table onto directus, I need to click and configure on every field. This inevitably takes me to the top of the page when I save and thus, a good chunk of the experience is me scrolling down where I left off before the save.

**Reply by rijkvanzanten** [reply]
Gotcha. There's currently not a way to do that in the app directly. You could use the `/fields` endpoint to programmatically do them all at once though, and then keep tweaking in the app

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
